### PR TITLE
changed rules for hex colors linter

### PR DESCRIPTION
**Pull request by matallo**
- lowercase
- short

so

```
color: #FF22EE;
```

wouldn't be valid, but would

```
color: #f2e;
```

based on personal preference, but serve this PR as a call for discussion, I can stick to other style though

@CartoDB/frontend

### CODE REVIEW

File: .scss-lint.yml [hunk 1]
```diff
@@ -42,11 +42,11 @@ linters:
     enabled: true
     present: true
   HexLength:
-    enabled: false
-    style: long
+    enabled: true
+    style: short
   HexNotation:
     enabled: true
```
Comment (xavijam) [inline .scss-lint.yml]: why not just disable this property?

File: .scss-lint.yml [hunk 2]
```diff
@@ -42,11 +42,11 @@ linters:
...
     present: true
   HexLength:
-    enabled: false
-    style: long
+    enabled: true
```
Comment (xavijam) [inline .scss-lint.yml]: same here

**Comment by viddo** [comment]
:+1:

File: .scss-lint.yml [hunk 3]
```diff
@@ -65,7 +65,7 @@ linters:
     severity: warning
   LeadingZero:
     enabled: true
-    style: include_zero
+    style: exclude_zero
```
Comment (xavijam) [inline .scss-lint.yml]: Uhm... I'd like to have the zero :DDD

Comment (matallo) [inline .scss-lint.yml]: First-come, first-served, I had to try (I'll stick to your convention)

File: .scss-lint.yml (hunk 1 above)
Comment (matallo) [inline .scss-lint.yml]: the idea of having this linter is make our code consistent, isn't it?

**Comment by matallo** [comment]
are we ok with these small changes?

**Comment by xavijam** [comment]
:+1:

**Pull request merged**